index.bs

```diff
@@ -3246,15 +3295,19 @@ Note: The {{UserVerificationRequirement}} enumeration is deliberately not refere
 
 <div dfn-type="enum-value" dfn-for="UserVerificationRequirement">
     :   <dfn>required</dfn>
-    ::  This value indicates that the [=[RP]=] requires [=user verification=] for the operation and will fail the operation if the
+    ::  The [=[RP]=] requires [=user verification=] for the operation and will fail the operation if the
         response does not have the [=UV=] [=flag=] set.
+        The [=client=] MUST return an error if [=user verification=] cannot be performed.
+
+        Note: The value of the [=UV=] [=flag=] might not be reliable without a trusted [=authenticator=] [=attestation=].
 
     :   <dfn>preferred</dfn>
-    ::  This value indicates that the [=[RP]=] prefers [=user verification=] for the operation if possible, but will not fail the
+    ::  The [=[RP]=] prefers [=user verification=] for the operation if possible, but will not fail the
         operation if the response does not have the [=UV=] [=flag=] set.
+        For example, the client SHOULD guide the user through setting up [=user verification=] if needed.
```

Comment:
I'm a bit confused by this last part, in the context of assertion generation. If a credential has already been established (without UV), what value is there in an RP requesting uv=required or uv-preferred and having the client go through a setup process during navigator.credentials.get()?

Reply:
To my way of thinking the usefulness of UV is the concept that the user who created the credential is the same user who uses it for authentication. This seems to be backed up by the observation that most authenticators "erase all their creds", or equivalent, when the UV mechanism is reset.

Reply:
You're right that UV only really becomes meaningful the second time you see it for a given credential, but that doesn't mean that the first time must necessarily have been the registration ceremony. If it was, then that of course covers the whole lifetime of the credential, but you could also imagine "enabling" UV at a later time which would then cover assertions made after that point.

The definition of [User Verification](https://www.w3.org/TR/2021/REC-webauthn-2-20210408/#user-verification) also slightly touches on this:

>Note that user verification does not give the Relying Party a concrete identification of the user, but when 2 or more ceremonies with user verification have been done with that credential it expresses that it was the same user that performed all of them. [...]

Reply:
The problem is if you enable UV later, you add inconsistency and potential confusion to the user. There are also some potential abuse vectors there where say a CTAP device on firefox (that only supports CTAP 1) will never ask for UV, but when you use it in chrome (which does CTAP2) does. Someone could add a password without the users knowledge on first use, and then that would mean the credential is locked out.

The other issue is that it means that in firefox the expectation that the credential is verified goes away, making it effectively disabled (especially weth uv preferred).

So IMO to keep a consistent behaviour for both the IDM which may have security policies in place and a need to understand the state of credentials, but also from a human interaciton point of view to maintain consintent and predicable behaviour, the UV state at registration is really how the device should be "forever" treated.

Reply:
@equalsJeffH Sure. I'd just like to note that this addition was based on the similar recommendation on [`ResidentKeyRequirement/preferred`](https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/1621.html#dom-residentkeyrequirement-preferred).

Reply:
Resolving this conversation as the "guiding" sentence has been removed.